### PR TITLE
Landing Page | style above the fold

### PR DESCRIPTION
**Pull request by afonsograca**
- created a toast component
- allowed the background to be displayed behind the button

### CODE REVIEW

File: src/routes/+page.svelte [hunk 1]
```diff
@@ -8,6 +8,8 @@
 	import AnimatedCarousel from '$lib/components/AnimatedCarousel/AnimatedCarousel.svelte'
 	import Text from '$lib/components/Text.svelte'
   import Button from '$lib/components/Button/Button.svelte'
+  import Toast from '$lib/components/Toast/Toast.svelte'
```
Comment (dinocosta) [inline src/routes/+page.svelte:11]: Weird indentation? 🤔

File: src/routes/+page.svelte [hunk 2]
```diff
@@ -45,16 +47,19 @@
...
 			<Text type="subhead" color="darkSecondary" class="sm:max-w-[55%] text-center">
 				Our AI Partnerships Intelligence Platform analyzes sales calls to uncover partner
 				opportunities. Our insights and workflows empower you to boost revenue and close deals faster
 				through partnerships.
 			</Text>
-			<Button type="primary" href={getEarlyAccessHref}>{getEarlyAccessText}</Button>
+		</div>
+		<div class="flex flex-col relative">
+			<img class="hidden sm:block sm:-mt-6" src={HeroBackground} alt="Background logo" />
+			<Button class="place-self-center sm:z-10 sm:absolute" type="primary" href={getEarlyAccessHref}>{getEarlyAccessText}</Button>
```
Comment (dinocosta) [inline src/routes/+page.svelte:63]: I believe you can get rid of the `sm:` here for both the `z-*` and `absolute` classes, if I'm not mistaken it should have no effect, as it should be basically stating "for every screen with a width bigger than or equal to 640 pixels", which includes all the other breakpoints.

More on that on Tailwind's documentation → [TailwindCSS Breakpoint Docs](https://tailwindcss.com/docs/screens).

**Review by dinocosta — CHANGES_REQUESTED** [review]
Can you also delete the `navSize` variable that is now unused as well as removing the `style` on this page (as well as removing the `id="hero"`)? 👍 

I'm seeing a duplicate on the hero background, and I believe it's because we still have the `<style>` tag at the bottom, setting the hero section's background

SCR-20240811-cnhf

File: src/routes/+page.svelte (hunk 1 above)
Comment (afonsograca) [reply src/routes/+page.svelte:11]: this must be a GH glitch, I'm not seeing any formatting issues

File: src/routes/+page.svelte (hunk 2 above)
Comment (afonsograca) [reply src/routes/+page.svelte:63]: I believe you addressed this already

**Pull request merged**